src/main/java/com/library/model/entity/Author.java

```diff
@@ -0,0 +1,40 @@
+package com.library.model.entity;
+
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.ManyToMany;
+
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.time.LocalDate;
+import java.util.List;
+
+@Entity
+@Getter
+@Setter
+@NoArgsConstructor
+public class Author extends ParentEntity {
+
+    @NotBlank
+    @Column
+    private String name;
+
+    @NotBlank
+    @Column
+    private String surname;
+
+    @NotNull
+    @Column(name = "date_of_birth")
```

Comment:
Still used unnecessary renaming of column.
No need to use literary naming.

Not "surname_of_author" or "book_publisher", as like as not "date_of_birth".

Reply:
ok